### PR TITLE
chore: Update dependencies

### PR DESCRIPTION
**Pull request by Xotic750**

### CODE REVIEW

**Comment by Xotic750** [comment]
Only eslint 5 tests pass, I added ajv back into the mix but that didn't help them.

File: package.json [hunk 1]
```diff
@@ -5,7 +5,8 @@
   "main": "dist/lib/rule-finder.js",
   "scripts": {
     "cover": "nyc --reporter=lcov --reporter=text npm test",
-    "lint": "xo",
+    "lint": "eslint --ext .js,.json .",
```
Comment (ljharb) [inline package.json]: switching eslint to xo seems like a good but separate PR?

File: package.json [hunk 2]
```diff
@@ -5,7 +5,8 @@
...
+    "lint-fix": "npm run lint -- --fix",
```
Comment (ljharb) [inline package.json]: not sure this script is needed at all; you can always stick a `--` on the end of the `lint` script

File: package.json [hunk 3]
```diff
@@ -30,34 +31,34 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "cliui": "^3.2.0",
-    "eslint-rule-documentation": "^1.0.0",
-    "path-is-absolute": "^1.0.1",
-    "which": "^1.2.8",
-    "window-size": "0.3.0",
-    "yargs": "^8.0.1"
+    "cliui": "^5.0.0",
```
Comment (ljharb) [inline package.json]: let's not upgrade things past major versions; that's far too breaking a change to be useful.

File: package.json (hunk 1 above)
Comment (Xotic750) [reply package.json]: I can do that.

File: package.json (hunk 2 above)
Comment (Xotic750) [reply package.json]: That was a lazy copy/paste.

File: package.json (hunk 3 above)
Comment (Xotic750) [reply package.json]: Went to v4 as not to skip a major.

Comment (ljharb) [reply package.json]: let's keep it on v3.

Comment (Xotic750) [reply package.json]: aha, ok

**Comment by Xotic750** [comment]
Meh, still only passes eslint 5.

**Comment by ljharb** [comment]
This module doesn't yet support eslint 6; see #304.

File: .travis.yml [hunk 4]
```diff
@@ -1,5 +1,7 @@
 sudo: false
 language: node_js
+cache:
+  npm: false
```
Comment (ljharb) [inline .travis.yml]: O.o?

Comment (Xotic750) [reply .travis.yml]: yep, that seems to have fixed the testing on travis

Comment (Xotic750) [reply .travis.yml]: I guess it was my original over eagerness with updating all those modules :P

Comment (Xotic750) [reply .travis.yml]: Just fails on node 4 now

Comment (Xotic750) [reply .travis.yml]: huh, remove that and now it has cached again, grr

**Comment by Xotic750** [comment]
So there we have it, bare minimum changes that will squelch that security message. If you are happy with this then I will close this PR and do a nice clean one instead. I will submit some more smaller PR's for other updates.

**Comment by Xotic750** [comment]
Closing in favour of https://github.com/sarbbottam/eslint-find-rules/pull/309

**Pull request closed without merging**